spec/models/supertype_spec.rb

```diff
@@ -6,8 +6,6 @@
       describe "Supertype #{supertype.id}" do
         it "has the required attributes for #{supertype.id}" do
           expect(supertype.id).to_not be_blank
-          expect(supertype.label).to_not be_blank
```

Comment:
I think we're losing something here in that we expect the associated locale keys to exist. Perhaps we can test for that instead?

Reply:
Have added this in :)